File: TIC10002-Py/src/uff/ic/lleme/tic10002/trabalhos/20171/viniciusferreira/Venda.py
class Venda:
    def __init__(self, id=None, filial=None, data=None, vendedor=None, total=None):
        self.id = id
        self.filial = filial
        self.data = data
        self.vendedor = vendedor
        self.total = total

    def getVenda(self):
        return [self.id, self.filial, self.data, self.vendedor, self.total]

File: TIC10002/src/main/java/uff/ic/lleme/tic10002/trabalhos/_20171/emersonsouza/Venda.java
package uff.ic.lleme.tic10002.trabalhos._20171.emersonsouza;

public class Venda {

    public int filial;
    public int anoMes;
    public int codVendedor;
    public double totalVendido;

    public Venda(String filial, String ano_mes, String cod_vendedor, String total_vendido) {
        this.filial = Integer.parseInt(filial);
        this.anoMes = Integer.parseInt(ano_mes);
        this.codVendedor = Integer.parseInt(cod_vendedor);
        this.totalVendido = Double.parseDouble(total_vendido);
    }

    public void printVenda(Venda dado) {
        System.out.println("Filial: " + this.filial + "\n"
                + "Data: " + this.anoMes + "\n"
                + "Código do Vendedor: " + this.codVendedor + "\n"
                + "Total Vendido: " + this.totalVendido + "\n");
    }

    public int getValor(int value) {
        if (value == 1)
            return filial;
        else if (value == 2)
            return anoMes;
        else
            return 0;
    }
}
